torchao/dtypes/affine_quantized_tensor.py

```diff
@@ -270,9 +290,11 @@ def from_hp_to_floatx(
         cls,
         input_float: torch.Tensor,
         block_size: Tuple[int, ...],
-        target_dtype: torch.dtype = torch.float8_e4m3fn,
+        target_dtype: torch.dtype,
+        scale_dtype: Optional[torch.dtype] = None,
         layout_type: LayoutType = PlainLayoutType(),
```

Comment:
will fp8 work with `PlainLayoutType` or will it always be `Float8LayoutType`?

Reply:
Yeah let me update the defaults